### DISCUSSION TITLE
Adding other Redhat's certified CNI plugins to e2e tests

### DISCUSSION THREAD

**Discussion by alibo** [okd-project/okd#988] [Ideas]
I'd like to share the idea of testing other Redhat's certified CNI plugins in e2e tests (especially for the upgrades):

![Screen Shot 2021-11-24 at 13 06 03](https://user-images.githubusercontent.com/5338482/143212772-a62e3bfc-585f-4d27-9224-f4abe9a7ce74.jpg)
https://access.redhat.com/articles/5436171

for example, cilium supports Openshift and OKD and it also has its own testing tools (which can be used in the e2e tests)

- https://cilium.io/blog/2021/04/19/openshift-certification
- https://docs.cilium.io/en/v1.10/gettingstarted/k8s-install-openshift-okd/
- https://github.com/cilium/cilium-olm/
- https://github.com/cilium/cilium/tree/master/examples/kubernetes/connectivity-check

WDYT?

**Comment by vrutkovs** [comment]
I don't mind setting up CI jobs if we find someone to look after these and fix issues

**Reply by alibo** [reply]
I wish I could do it, but apparently, your company doesn't allow me to contribute! so I hope you can find someone else :) 
- https://github.com/openshift/enhancements/pull/855#issuecomment-973880734
- https://github.com/openshift/cluster-ingress-operator/pull/634#issuecomment-973880888

(the MR author is my colleague)
